Modules/SubModule_Import/xASL_imp_NII2BIDS_RunAnat.m

```diff
@@ -56,6 +56,12 @@ function xASL_imp_NII2BIDS_RunAnat(imPar, bidsPar, studyPar, subjectSessionLabel
             % Create the anatomical directory
             xASL_adm_CreateDir(anatOutDirLabel);
 
+            % Current scan name
+            [~, scanName] = xASL_fileparts([anatOutLabel,'_' iAnatType{1}]);
+            
+            % Print anat name
+            fprintf('Scan %s ...\n', scanName);
+            
             % Move the NiFTI file
             anatNiiPath = [anatOutLabel,'_' iAnatType{1} '.nii.gz'];
```

Comment:
Also here the `,` is confusing. Are you sure both shouldn't have `fullfile`?

Reply:
We can optimize this, but that's really just cosmetics IMHO

Reply:
yes - here I agree that you should either have `','` or `' '` everywhere. No `fullfile` that's just filename being constructed, not path

Reply:
Check line 37 of that function - the most of the path is already constructed using `fullfile`, so this is just a filename postfix - `fullfile` cannot be used here anymore.

Reply:
`,` fixed throughout the code